governance/proposals/up/002-change-voting-duration.js

```diff
@@ -0,0 +1,80 @@
+const { ethers } = require('hardhat')
+const { base } = require('@unlock-protocol/networks')
+const { getNetwork } = require('@unlock-protocol/hardhat-helpers')
+
+const VOTING_DELAY = 4 * 24 * 60 * 60 // 4 days in seconds
+const VOTING_PERIOD = 6 * 24 * 60 * 60 // 6 days in seconds
+const MIN_DELAY = 2 * 24 * 60 * 60 // 2 days in seconds
+const BASE_TIMELOCK_ADDRESS = '0xB34567C4cA697b39F72e1a8478f285329A98ed1b'
+const BASE_GOVERNOR_ADDRESS = '0x65bA0624403Fc5Ca2b20479e9F626eD4D78E0aD9'
```

Comment:
There is now a better way to get the gov address - from the dao section in the networks package.

Reply:
Please show me the better way to get the governor address you are referring to, maybe I'm missing something but I couldn't the governor address in the networks package, I only found an UnlockAddress property.

Reply:
![local-unlock-networks-log](https://github.com/user-attachments/assets/ff5cf6ba-bc4c-4227-8a68-f60683be33ef)  

This is what it looks like when I log to the console.  I pulled latest changes, deleted node modules and cleaned the cache but still get the same thing, no dao section available.  

![unlock-networks-package](https://github.com/user-attachments/assets/7a067380-fc59-458e-91a4-c97fe9dd24a2)  

However, if I go to the folder [unlock/packages/networks/src/networks/base.ts](https://github.com/unlock-protocol/unlock/blob/5580b88715c348444a9a6586d28c570a15f3f4fd/packages/networks/src/networks/base.ts#L18) after pulling the latest changes I see the dao section there. Not sure why that's happening.  
This is how I import the package:  
```
const {base }= require('@unlock-protocol/networks')

```

Reply:
you may need to run `yarn && yarn build` from the root of the monorepo so the packages are rebuilt